### DISCUSSION TITLE
nnn exit codes (filepicker)

### DISCUSSION THREAD

**Discussion by vlappa** [jarun/nnn#1846] [Ideas]
Another idea from fzf. It has several exit codes which could be useful for nnn when in filepicker mode, useful for scripting.

From fzf:
```EXIT STATUS
       0      Normal exit
       1      No match
       2      Error
       130    Interrupted with CTRL-C or ESC

**Comment by N-R-K** [comment]
Don't think there's any plans for it. Keep in mind that `nnn` is a file-manager first and foremost. The picker mode was an afterthought. Whereas `fzf` was designed ground up as a picker.
